Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -108,10 +110,18 @@ public extension CBPurchase {
             retrieveProducts()
         }
     }
-
+        
+    func purchaseNonSubscriptionProduct(product: CBProduct, customer : CBCustomer? = nil ,productType : ProductType? = nil,completion handler: @escaping ((_ result: Result<(NonSubscription), Error>) -> Void)) {
```

Comment:
Minor: Remove the brackets for `(NonSubscription)`

Reply:
updated